MSDK build have a warning with gcc-9.1.0 version

I got following warning while building MSDK with gcc 9.1 on ubuntu 18.04.
This warning will threat as error because of -Werror flag in CI build configuration.

```
In file included from /localdisk/bb/mediasdk_gcc_9.1/build_dir/repos/MediaSDK/samples/sample_decode/src/pipeline_decode.cpp:21:
/localdisk/bb/mediasdk_gcc_9.1/build_dir/repos/MediaSDK/samples/sample_decode/include/pipeline_decode.h: In constructor ‘sInputParams::sInputParams()’:
/localdisk/bb/mediasdk_gcc_9.1/build_dir/repos/MediaSDK/samples/sample_decode/../sample_common/include/sample_defs.h:168:78: warning: ‘void* memset(void*, int, size_t)’ clearing an object of non-trivial type ‘struct sInputParams’; use assignment or value-initialization instead [-Wclass-memaccess]
  168 | #define MSDK_ZERO_MEMORY(VAR)                    {memset(&VAR, 0, sizeof(VAR));}
      |                                                                              ^
/localdisk/bb/mediasdk_gcc_9.1/build_dir/repos/MediaSDK/samples/sample_decode/include/pipeline_decode.h:132:9: note: in expansion of macro ‘MSDK_ZERO_MEMORY’
  132 |         MSDK_ZERO_MEMORY(*this);
      |         ^~~~~~~~~~~~~~~~
In file included from /localdisk/bb/mediasdk_gcc_9.1/build_dir/repos/MediaSDK/samples/sample_decode/src/pipeline_decode.cpp:32:
/localdisk/bb/mediasdk_gcc_9.1/build_dir/repos/MediaSDK/samples/sample_decode/include/pipeline_decode.h:74:8: note: ‘struct sInputParams’ declared here
   74 | struct sInputParams
      |        ^~~~~~~~~~~~
[ 11%] Building CXX object samples/sample_decode/CMakeFiles/sample_decode.dir/src/sample_decode.cpp.o
```

Full log here: [build.log](https://github.com/Intel-Media-SDK/MediaSDK/files/3431502/build.log)

--- a/samples/sample_decode/include/pipeline_decode.h
+++ b/samples/sample_decode/include/pipeline_decode.h
@@ -127,10 +127,6 @@ struct sInputParams
     msdk_char     strDstFile[MSDK_MAX_FILENAME_LEN];
     sPluginParams pluginParams;
 
-    sInputParams()
-    {
-        MSDK_ZERO_MEMORY(*this);
-    }
 };
 
 template<>struct mfx_ext_buffer_id<mfxExtMVCSeqDesc>{

--- a/samples/sample_decode/src/sample_decode.cpp
+++ b/samples/sample_decode/src/sample_decode.cpp
@@ -681,7 +681,7 @@ int _tmain(int argc, TCHAR *argv[])
 int main(int argc, char *argv[])
 #endif
 {
-    sInputParams        Params;   // input parameters from command line
+    sInputParams        Params = {};   // input parameters from command line
     CDecodingPipeline   Pipeline; // pipeline for decoding, includes input file reader, decoder and output file writer
 
     mfxStatus sts = MFX_ERR_NONE; // return value check

